tensorflow_datasets/testing/test_data/fake_examples/spine_web/test_angles.csv

```diff
@@ -0,0 +1,128 @@
+43.423,15.872,40.687
```

Comment:
@chriszhou0916 are these random or from the real dataset?

Reply:
I changed it in the new commit. They're now randomized and only one row in each.